src/stories/index.stories.js

```diff
@@ -113,13 +141,55 @@ export const backgroundImagesWithFade = () =>
     autoplay={ true }
     arrows={ true }
   >
-    <div style={{ backgroundImage: 'url(http://picsum.photos/600/300)', backgroundSize: 'cover', height: '100%', width: '100%' }}/>
-    <div style={{ backgroundImage: 'url(http://picsum.photos/650/300)', backgroundSize: 'cover', height: '100%', width: '100%' }}/>
-    <div style={{ backgroundImage: 'url(http://picsum.photos/675/300)', backgroundSize: 'cover', height: '100%', width: '100%' }}/>
-    <div style={{ backgroundImage: 'url(http://picsum.photos/700/300)', backgroundSize: 'cover', height: '100%', width: '100%' }}/>
-    <div style={{ backgroundImage: 'url(http://picsum.photos/750/300)', backgroundSize: 'cover', height: '100%', width: '100%' }}/>
-    <div style={{ backgroundImage: 'url(http://picsum.photos/725/300)', backgroundSize: 'cover', height: '100%', width: '100%' }}/>
-    <div style={{ backgroundImage: 'url(http://picsum.photos/625/300)', backgroundSize: 'cover', height: '100%', width: '100%' }}/>
+    <div style={{
+      backgroundImage: 'url(http://picsum.photos/600/300)',
+      backgroundSize:
+      'cover',
```

Comment:
nitpick: why do these need to be on a newline? Seems more appropriate to just have `backgroundSize: 'cover'`?

Reply:
good catch. they don't.